src/ant/README.md

```diff
@@ -0,0 +1,37 @@
+# Antenna Rule Checker
+
+This tool checks antenna violations of a design in opendb, a report
+will be generated to indicate violated nets. APIs are provided to help fix antenna violation during the diode insertion flow in globalroute:
+
+## Antenna Check Commands
+
+---
+
+ - `check_antennas`: check antenna violations on all nets and generate a report
+   - -report_filename: specify the file name path where to save the report
+   - -report_violating_nets: provides a summary of the violated nets 
+
+## Antenna Report Example
+---
+
+This is an example of the detailed and simple reports of the antenna checker:
+
+| <img src="./doc/images/ant_report_print.png" width=400px> |  
```

Comment:
I would strongly discourage having pictures of text in documentation. Use a formatted code block in the README instead.

Reply:
I agree

Reply:
Sure, will work on that.